src/autopas/containers/linkedCells/traversals/LCC04Traversal.h

```diff
@@ -46,7 +46,7 @@ class LCC04Traversal : public C08BasedTraversal<ParticleCell, PairwiseFunctor, d
                                                                                  interactionLength, cellLength),
         _cellOffsets32Pack(computeOffsets32Pack()),
         _cellHandler(pairwiseFunctor, this->_cellsPerDimension, interactionLength, cellLength, this->_overlap),
-        _end(utils::ArrayMath::subScalar(utils::ArrayUtils::static_cast_array<long>(this->_cellsPerDimension), 1l)) {}
+        _end(utils::ArrayMath::operator-(utils::ArrayUtils::static_cast_array<long>(this->_cellsPerDimension), 1l)) {}
```

Comment:
I'm not really convinced that this change improves readability.

Reply:
done in [fcbb30b](https://github.com/AutoPas/AutoPas/pull/735/commits/fcbb30b047a00acac67d3090e735f6141ffc5da8).